src/main/java/amogus/Amogus.java

```diff
@@ -0,0 +1,55 @@
+package amogus;
+
+import command.Command;
+import tasks.TaskList;
+
+import java.io.IOException;
+
+public class Amogus {
+
+    static final String NAME = "amogus";
+    private static final String path = "./data/Tasks.TaskList.txt";
```

Comment:
Perhaps you could capitalize this variable name as well?
```suggestion
    private static final String PATH = "./data/Tasks.TaskList.txt";
```